ViewControllers/MostPopularViewController.swift

```diff
@@ -86,68 +91,28 @@ class MostPopularViewController: UIViewController, UICollectionViewDelegate {
         }
     }
     
-    @objc func seeMoreTapped(_ sender: UIButton) {
+    @objc internal func seeMoreTapped() {
```

Comment:
I'm curious why we went for a `internal` here, we can discuss in the meeting Today